dz2/MainClass.java

```diff
@@ -0,0 +1,196 @@
+
+        printArray(arrayThree);
+
+        arrayCheckSix(arrayThree);
+
+        printArray(arrayThree);
+
+        // 4 Создать квадратный двумерный целочисленный массив (количество строк и столбцов
+        // одинаковое), и с помощью цикла(-ов) заполнить его диагональные элементы единицами;
+
+        int[][] arrayFour = new int[9][9];
+
+        for (int i = 0; i < arrayFour.length; i++) {
+            for (int j = 0; j < arrayFour[i].length; j++) {
+                if (i == j) {
+                    arrayFour[i][j] = 1;
+                }
+                if (i + j == arrayFour[i].length - 1) {
+                    arrayFour[i][j] = 1;
+                }
+            }
+        }
+
+        printDoubleArray(arrayFour);
+
+        // 5 ** Задать одномерный массив и найти в нем минимальный и максимальный элементы (без
+        //  помощи интернета);
+
+        int[] arrayFive = {3, 8, 12, 6, 7, 12, 5, 8, 22, 9, 34, 12, 7, 9, 36, 44, 7, 59, -12, -36, 99, 158};
+
+        sortMaxMin(arrayFive);
+
+        // 6 ** Написать метод, в который передается не пустой одномерный целочисленный массив,
+        // метод должен вернуть true если в массиве есть место, в котором сумма левой и правой части
+        // массива равны. Примеры: checkBalance([1, 1, 1, || 2, 1]) → true, checkBalance ([2, 1, 1, 2, 1]) →
+        // false, checkBalance ([10, || 10]) → true, граница показана символами ||, эти символы в массив не
+        // входят.
+
+        int[] arraySix = {3, 3, 13, 6, 6, 4, 1, 1, 22, 15, 28, 4, 6, 14, 22};
+
+        System.out.println(checkBalance(arraySix));
+    }
+
+    public static void inverseArray(int[] arr) {     //Метод инвертирует значение ячейки массива с 0 на 1 или с 1 на 0
+        for (int number : arr) {
+            System.out.print(number);
+        }
+        System.out.println();
+
+
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] == 0) {
+                arr[i] = 1;
+            } else arr[i] = 0;
+        }
+
+        for (int number : arr) {
+            System.out.print(number);
+        }
+        System.out.println();
+        System.out.println();
+    }
+
+    public static void printArray(int[] arr) {      // Метод выводит на печать интовый одномерный массив
+        for (int o : arr) {
+            System.out.print(o + " ");
+        }
+        System.out.println();
+        System.out.println();
+    }
+
+    public static void printDoubleArray(int[][] arr) {
+        for (int[] ints : arr) {
+            for (int anInt : ints) {
+                System.out.print(anInt + " ");
+            }
+            System.out.println();
+        }
+        System.out.println();
+    }
+
+    public static void arrayCheckSix(int[] arr) {     // Метод проверяет все ячейки массива и те которые меньше 6-ти умножает на 2
+        for (int i = 0; i < arr.length; i++) {
+            if (arr[i] < 6) arr[i] *= 2;
+        }
+    }
+
+    public static void sortMaxMin(int[] arr) {
```

Comment:
Название должно отражать то, что метод делает.
Он ничего не сортирует.
В данном случае лучше наверное printMaxAndMin или типа того
```suggestion
    public static void sortMaxMin(int[] arr) {
```